src/DetailsView/components/no-content-available.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as Markup from 'assessments/markup';
+import { NamedFC } from 'common/react/named-fc';
+import { productName } from 'content/strings/application';
+import * as React from 'react';
+
+import { targetPageClosed } from './target-page-closed-view.scss';
```

Comment:
Should we update the CSS class name to match the component name?
```suggestion
import { noContentAvailable } from './no-content-available-view.scss';
```

Reply:
Good catch! Updated.